openfmri/ds000030/model_with_rt.json

```
@@ -0,0 +1,111 @@
+{
+    "name": "ds000030_bart_with_rt",
+    "description": "model for balloon analog risk task",
+    "input": {
+        "task": "bart"
+    },
+    "blocks": [
+        {
+            "level": "run",
+            "transformations": [
+                {
+                    "name": "factor",
+                    "input": [
+                        "trial_type",
+                        "action"
+                    ]
+                },
+                {
+                    "name": "and",
+                    "input": [
+                        "trial_type.BALOON",
+                        "action.ACCEPT"
+                    ],
+                    "output": [
+                        "accept"
+                    ]
+                },
+                {
+                    "name": "and",
+                    "input": [
+                        "trial_type.BALOON",
+                        "action.EXPLODE"
+                    ],
+                    "output": [
+                        "explode"
+                    ]
+                },
+                {
+                    "name": "or",
+                    "input": ["accept", "explode"],
+                    "output": "accept_or_explode"
+                },
+                {
+                     "name": "replace",
+                     "input": [ "accept_or_explode" ],
+                     "replace": { "0": "n/a"}
+                },
+                {
+                    "name": "drop_na",
+                    "input": ["accept_or_explode"]
+                },
+                {
+                    "name": "demean",
+                    "input": "reaction_time"
```

demean input should be an array.
```suggestion
                    "input": ["reaction_time"]
```